### ISSUE TITLE
Error when importing after upgrading to uSync Complete 12.2.1.1

### ISSUE THREAD

**Issue by jemayn** [KevinJump/uSync8#564]
**Describe the bug**

When running a schema import on a site with uSync it works just fine on usync.Complete version 12.2.0, and fails on 12.2.1.1 with unchanged schema files.

The error it hits is only in the browser:

/umbraco/backoffice/usync/usyncdashboardapi/ImportHandler

```json
{
    "ExceptionMessage": "An item with the same key has already been added. Key: b744f1eb-8b60-4d43-a52d-41b371f1c121",
    "ExceptionType": null,
    "StackTrace": null
}
```

The key corresponds to the GUID of a datatype, however the site doesn't have any datatypes with that ID already and it is only referenced in 3 usync files:
1. Its own datatype file
2. As a referenced datatype on a doctype file
3. As a rename of the name in a datatype rename file

After downgrading uSync to 12.2.0 again it works as expected and runs to completion

**Expected behavior**
I'd expect it to work

**About your Site (please complete the following information):**
 - Umbraco Version: 12.2.0
 - uSync Version: 12.2.0 / 12.2.1.1
 - Browser: Any

**Comment by KevinJump** [comment]
Hi, yes - looks like the new sort doesn't like the renames (specifically it tries to add them to the sorting collectiom when they are already there). 

Fix incoming ....

**Comment by KevinJump** [comment]
In v12.2.2 - https://www.nuget.org/packages/uSync/12.2.2

**Comment by jemayn** [comment]
Wow that was fast! Thanks Kevin 😊

**Comment by jemayn** [comment]
Hey Kevin,

Is it intentional that `uSync.Complete` is still only available as 12.2.1.1, so we need to have that and then install `uSync` 12.2.2 explicitly on the side?

Np if so just wondering if it's a mistake that the complete version isn't updated as that's all we had to install before 🙂

Can report back that the issue is solved for us once we install `uSync` v12.2.2
